### ISSUE TITLE
Clevis alternative

### ISSUE THREAD

**Issue by FadzLeb** [fox-it/linux-luks-tpm-boot#13]
Hello,

I am a windows user trying to switch to linux but I need TPM encryption for my purposes.

I have read your git page but I found through research something called clevis that links the TPM to the LUKS encryption:
https://github.com/latchset/clevis

So I have no idea if it is connected to your solution or it can make it easier but I think it's worth checking out

**Comment by morbitzer** [comment]
Hi, 
That does indeed sound relevant, thanks for pointing out the project. 

I guess you would still need to use TrustedGRUB2 to create the boot time measurement. This means you are still not able to use UEFI. 

Also, if I understood that correctly, clevis only seems to support TPM version 2. So you are required to have a TPM2 in your system, which is not very often the case, at least as far as I know.

**Comment by FadzLeb** [comment]
Yes from what I read that is the case it only supports TPM2, which is fine by me as for the boot measurement I have no clue since I am a Windows user mainly (The only reason I am still on Windows is the fact that I can't port my life's work to Linux) but I didn't read anything concerning the boot measurement.

But what surprises me is that these simple things (automated Bitlocking and other automated tools) don't exist in Linux. I am tackling this in my free time so if I found a simpler way to do this encryption I will contribute to this project.
